fuzz/fuzz_http3serverreq.cc

```diff
@@ -75,4 +97,4 @@ extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
   nghttp3_conn_del(conn);
 
   return 0;
-}
+}
```

Comment:
Looks like the line separator is missing.

Reply:
Fixed.